Kickstarter-iOS/Info.plist

```diff
@@ -62,6 +62,8 @@
 	<true/>
 	<key>NSAppTransportSecurity</key>
 	<dict>
+		<key>NSAllowsArbitraryLoads</key>
```

Comment:
Should this be added?

Reply:
Woops, no we don't wanna add that please 🙂

Reply:
Thanks!